codesec-agent/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
+_1.2.2_
+* Add envs to connect and scan deployments
```

Comment:
Let's try to be a bit more generous with the details we add to CHANGELOG, especially when dealing with customer-facing public repos.

For example in this case I'd write something like:

```
_1.2.2_
* Inject authentication (CSPM_URL) and API (AQUA_SERVER_URL) environment variables to deployment objects.
* Fix default value of "aquaServerUrl".
```